k8/images/geopandas/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# syntax = docker/dockerfile:1.1.7-experimental
+ARG FROM
+FROM ${FROM}
+
+WORKDIR /usr/src/app
+
+RUN cat /etc/os-release
+RUN apt-get install libgeos-dev
```

Comment:
I am not familiar with how the docker images are set up, but, *if* pip is using wheels, you don't need to install libgeos here (as it will be included in the shapely wheels). 
But maybe the base docker image is using Alpine? (in which case there are no wheels, and libgeos is indeed needed to install Shapely from source)

Reply:
I tried to do this because I was getting the error: `OSError: Could not find library geos_c or load any of its variants ['libgeos_c.so.1', 'libgeos_c.so']`

So, it seems that Shapely is not installing those binaries

Reply:
But still I haven't found the workaround for that in the docker image :/

Reply:
That sounds as it is trying to build from source, and not installing a wheel (in the wheels, GEOS is included, but when building from source the GEOS library needs to be installed already on the system).  
Not sure why it would not be using wheels, there are wheels available (unless the docker image is not manylinux compatible)

Reply:
I just send a new try, let me see if this error is fixed. I think that maybe as I'm doing a `pip install -e .` it is not including GEOS